test/e2e/systemd_activate_test.go

```diff
@@ -111,14 +85,14 @@ var _ = Describe("Systemd activate", func() {
 		}
 
 		podman := func(args ...string) *testUtils.PodmanSession {
```

Comment:
Should you need to repush for any reason, I think future maintainers might welcome a comment here explaining that we can't use the usual `podmanTest.Podman()` because we need to ensure that we invoke local-podman.

Or, on further thought, WDYT of using `podmanTest.Podman()`, and adding `SkipIfRemote()`?

Reply:
@vrothberg, #18083 has merged, so it's safe to proceed with this. I have a moderate preference for following up on my suggestion above (use `Podman`, add `SkipIfRemote`) but will not block on it.

Reply:
Done.  Note that the test already skips remote.  I moved it up a bit to make it easier to spot.